app/jobs/check_duns_job.rb

```diff
@@ -0,0 +1,7 @@
+class CheckDunsJob < ActiveJob::Base
+  queue_as :default
+
+  def perform(user)
+    SamAccountReckoner.new(user).set!
```

Comment:
defining jobs can be helpful if you have complex logic -- since all we're doing here is calling a method, we can instead call it where it was being called before and update to read:

`SamAccountReckoner.new(user).delay.set!`

this tells DJ to handle it in the background. see: https://github.com/collectiveidea/delayed_job#queuing-jobs